### PR TITLE
Cleanup LOGIN/STATUS packet handlers 

### PR DESCRIPTION
**Pull request by FlorianMichael**
Some combinations with the override boolean for ViaFabricPlus and other addons

### CODE REVIEW

**Comment by FlorianMichael** [comment]
I only added those which I would use but I can remove the ones you don't like.

File: common/src/main/java/com/viaversion/viaversion/protocols/v1_12_2to1_13/Protocol1_12_2To1_13.java [hunk 1]
```diff
@@ -173,11 +173,11 @@ protected void registerPackets() {
 
         WorldPacketRewriter1_13.register(this);
 
-        registerClientbound(State.LOGIN, ClientboundLoginPackets.LOGIN_DISCONNECT.getId(), ClientboundLoginPackets.LOGIN_DISCONNECT.getId(), wrapper -> {
+        registerClientbound(State.LOGIN, ClientboundLoginPackets.LOGIN_DISCONNECT.getId(), wrapper -> {
```
Comment (kennytv) [inline common/src/main/java/com/viaversion/viaversion/protocols/v1_12_2to1_13/Protocol1_12_2To1_13.java:176]: all of these can already just be replaced with
```java
registerClientbound(State.LOGIN, ClientboundLoginPackets.LOGIN_DISCONNECT, wrapper -> {
```
etc.

**Review by kennytv — CHANGES_REQUESTED** [review]
Actually, none of these are "standard" or needed for the main projects, helper methods are better added in the projects that need them

**Comment by FlorianMichael** [comment]
I'll keep those extra methods in ViaFabricPlus then; Resolved the suggestion

**Review by kennytv — APPROVED** [review]

**Pull request merged**